pkgs/by-name/md/mdbook-variables/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitLab,
+  versionCheckHook,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "mdbook-variables";
+  version = "0.2.4";
+
+  src = fetchFromGitLab {
+    owner = "tglman";
+    repo = "mdbook-variables";
+    rev = version;
+    hash = "sha256-whvRCV1g2avKegfQpMgYi+E6ETxT2tQqVS2SWRpAqF8=";
+  };
+
+  cargoHash = "sha256-uw1oWIoKi6qsObI4SkEiHwEj9QoxE9jufu9O+ZKM8DI=";
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  doInstallCheck = true;
+
+  meta = {
+    description = "mdBook preprocessor to replace values with env variables.";
```

```suggestion
    description = "MdBook preprocessor to replace values with env variables";
```
https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/meta.chapter.md#description-var-meta-description

---

Can skip first uppercase because mdBook is a project name